src/seedu/addressbook/Main.java

```diff
@@ -111,7 +111,7 @@ private CommandResult executeCommand(Command command)  {
             CommandResult result = command.execute();
             storage.save(addressBook);
             return result;
-        } catch (Exception e) {
+        } catch (Exception e) { // better way to create exception: create custom message
```

Comment:
Should remove unneeded comment, e.g. `// better way to create exception: create custom message`.